src/errorMessages.cpp

```diff
@@ -322,6 +322,15 @@ std::string ErrorMessages::WARNING_ONLY_CHILD(const DebugInfo& info,
     return errorMsg(0, ErrorLevel::WARNING, oss.str());
 }
 
+std::string ErrorMessages::WARNING_SOMA_SWC_2POINT(const morphio::Points& points) {
+    std::stringstream ss;
+    ss << "Warning: SWC soma has 2 points. Implicitly 3-points and 1-point SWC somas\n"
+          "are cylinders. 2-points SWC soma has no implicit well-established meaning.\n"
```

Comment:
that's not true; SWC somas are sections, which are well defined for 2 points - I'm in the process of getting a question ready for NEURON about this.

Reply:
Should I revert the PR?

Reply:
Sorry I did not know that ...

Reply:
Don't be sorry. It's not well established knowledge. For example here is the old discussion about SWC with Ted https://www.neuron.yale.edu/phpBB/viewtopic.php?p=18355#p18355. The main point:

>  There has never been an "official" definition of the SWC format, so it's hard to be dogmatic about this, but "one point somas" did not start to appear until some time (years?) after SWC was already in wide use.

Reply:
Agreed, no need to be sorry - this is a weird corner case.

IMO, it's fine to leave it until we have some clarity from what at least NEURON believes is true...which I'm trying to do.

Reply:
As I pointed in https://github.com/BlueBrain/MorphIO/issues/265#issuecomment-810383942 Neuron 7.8.2 implicitly drops the 2nd point and interprets 2-point soma as a single point one. Now I've installed the latest Neuron with #907 patch. I can confirm that the same example show that Neuron does not do such implicit processing anymore. 2-point soma stays as 2-point soma. All its points stay the same.